tests/classes/test_reportviews.py

```diff
@@ -0,0 +1,10 @@
+import pytest
+
+import xgi
+from xgi.exception import XGIError
+
+
+def test_edge_members(edgelist3):
+    H = xgi.Hypergraph(edgelist3)
+    assert H.edges.members() == [[1, 2, 3], [3, 4], [4, 5, 6]]
```

Comment:
I would add an assert to check a single ID, i.e. `H.edges.members(0)` and check that an Error is thrown in the `None` case and the single id case.

Reply:
I added the new test case for members(0). The case of members(None) is actually the default, which outputs all of them so it shouldn't give an error-1